src/VisualStudio/Core/Impl/Options/AbstractOptionPreviewViewModel.cs

```diff
@@ -173,7 +173,7 @@ public void UpdatePreview(string text)
                 LineSpan.FromBounds(startLine, endLine));
 
             var textView = _textEditorFactoryService.CreateTextView(projection,
-              _textEditorFactoryService.CreateTextViewRoleSet(PredefinedTextViewRoles.Analyzable));
+              _textEditorFactoryService.CreateTextViewRoleSet(PredefinedTextViewRoles.Interactive));
```

Comment:
Does this also need Analyzable as well? Otherwise we might lose classification or other tagging.